`exec` subcommand (#624)
ansible/ansible-navigator: docs/changelog-fragments.d/526.feature.md

> Add ability to run commands directly within an execution environment.

Reviewer: Per https://github.com/ansible/ansible-navigator/pull/641#discussion_r765392465, something like this may be better:
```suggestion
Added the ability to run arbitrary shell commands from the TUI directly
so they are invoked within an execution environment.
```

Added the ability to run ad-hoc commands directly within an execution
environment using mode `stdout`.

> 

Reviewer: How about being a bit clearer wrt what the code snippet below demonstrates?
```suggestion

Here is an example invocation:
```

> ```
> $ ansible-navigator exec --mode stdout

Reviewer: There are a few things to note here:
1) It's usually useful to select the highlighting language for code blocks to avoid ambiguity. This has a better chance of being parsed with the desired lexer. For snippets of commands having `$` prompt, it should usually be `console` or its alias `shell-session`.
2) The way `console` works is that it uses different colors for the commands with prompts and their output.
3) markdownlint will probably be unhappy about having a prompt+command with no output (I've hit that rule recently). Although, you won't see this problem until the release (when this gets into `CHANGELOG.md`) because the notes are excluded from linting.
4) If you only intend to demonstrate a command, you could use `bash`/`sh` instead and drop the prompt.
````suggestion
```console
$ ansible-navigator exec --mode stdout
[example output]
````

Author: I'm gonna leave it without a prompt or the output of the command.  The reason I added this wan't really so it showed the sample output, but to show how the command looked with `exec` and the ability to cut-paste from the change log...  My theory, completely unproven, is that someone might be more likely to try it out if they could cut and paste :)

Reviewer: > My theory, completely unproven, is that someone might be more likely to try it out if they could cut and paste :) (or use the clipboard icon when it's available)

AFAIR that clipboard icon Sphinx extension actually removes the prompt on copy and it's configurable what it should remove. Needs some testing, though, because I'm not sure how it's configured currently.

Added the ability to run ad-hoc commands directly within an execution
environment using mode `stdout`.

Here is an example invocation:

> ```console

Reviewer: Only use `console` or `shell-session` if you have a prompt with the command, followed by its output. If there's no prompt, don't use this lexer because it'll treat the content (well, anything that does not start with a prompt or a comment-hash) as "console output". Instead, you could choose something like `bash`.
```suggestion
```bash
```

These names are coming from https://pygments.org/docs/lexers/#pygments.lexers.shell.BashSessionLexer.
